client/src/main/java/io/pravega/client/netty/impl/ConnectionPoolImpl.java

```diff
@@ -263,16 +261,18 @@ private SslContext getSslContext() {
             log.debug("Setting up an SSL/TLS Context");
             try {
                 SslContextBuilder clientSslCtxBuilder = SslContextBuilder.forClient();
+
                 if (Strings.isNullOrEmpty(clientConfig.getTrustStore())) {
-                    clientSslCtxBuilder = clientSslCtxBuilder.trustManager(FingerprintTrustManagerFactory
-                                                                                   .getInstance(FingerprintTrustManagerFactory.getDefaultAlgorithm()));
-                    log.debug("SslContextBuilder was set to an instance of {}", FingerprintTrustManagerFactory.class);
+                    log.debug("Client truststore wasn't specified.");
+                    File clientTruststore = null; // variable for disambiguating method call
+                    clientSslCtxBuilder.trustManager(clientTruststore);
```

Comment:
ok, setting trustManager(null) uses the system default trust store

Reply:
You can also do `(File)null` so you don't have to declare a new variable just for this.